docs/source/changelog.rst

```diff
@@ -1 +1 @@
-.. include:: ../../CHANGES.rst
\ No newline at end of file
+.. include:: ../../CHANGES.rst
```

Comment:
Note to self: It's not clear from the GitHub diff, but this change is adding a missing trailing newline.